docs/pipeline/actions/piimasking.mdx

```diff
@@ -0,0 +1,65 @@
+---
+title: "Pii Masking"
+sidebarTitle: "Pii Masking"
+---
+
+The "Pii Masking" Odigos Action masks span attribute values that match a blocked value list.
+
+### Use Cases
+
+- Prevent sensitive fields from accidentally leaking into traces
+- Ensure compliance with legal, privacy, or security requirements
+
+
+### Mechanism
```

Comment:
You can also mention that the matching is done with regular expression, which describes the mechnism that is being used